REFACTOR: Import gds

gkorompi opened the pull request.

hui-zhou-a reviewed (changes requested):
Please explain why do you delete import_layout_pcb?

gkorompi commented:
> Please explain why do you delete import_layout_pcb?

Hey @hui-zhou-a, import gds was not working properly, not importing tech files and ControlFile class has missing properties. Instrad, I implemented importing gds files in import_layout_pcb, and changed the name to be more generic

hui-zhou-a commented:
> > Please explain why do you delete import_layout_pcb?
> 
> Hey @hui-zhou-a, import gds was not working properly, not importing tech files and ControlFile class has missing properties. Instrad, I implemented importing gds files in import_layout_pcb, and changed the name to be more generic

@gkorompi you should not delete any existing method. It may break the customer's code.

hui-zhou-a reviewed (dismissed):
You should no rename/delete any existing method. 

As an alternative, you can use deprecation warning. This is an example https://github.com/ansys/pyedb/blob/cb5089fd6ad0189844417dc1a3db003c3d0ef475/src/pyedb/dotnet/edb.py#L780

svandenb-dev reviewed (approved):
LGTM thanks for your work !!

svandenb-dev commented:
> Please explain why do you delete import_layout_pcb?

I have added the deprecation. @gkorompi, @hui-zhou-a is right do not rename without deprecating to avoid breaking users workflow.

svandenb-dev reviewed (approved):
LGTM
